test/e2e/bgppolicy_test.go

```diff
@@ -130,77 +123,193 @@ func TestBGPPolicy(t *testing.T) {
 	require.NotEqual(t, "", svcClusterIP.Spec.ClusterIP, "ClusterIP should not be empty")
 	clusterIP := svcClusterIP.Spec.ClusterIP
 
-	t.Log("Create a test BGPPolicy selecting all Nodes as well as advertising ClusterIPs and Pod CIDRs")
-	bgpPolicy := &crdv1alpha1.BGPPolicy{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: bgpPolicyName,
-		},
-		Spec: crdv1alpha1.BGPPolicySpec{
-			NodeSelector: metav1.LabelSelector{
-				MatchLabels: map[string]string{},
+	t.Run("One BGPPolicy applied to all Nodes", func(t *testing.T) {
+		const bgpPolicyName = "test-policy-alfa"
```

Comment:
I think you mean "alpha" :P

Reply:
Yes, it should be `alpha`. I considered creating multiple policies with the names `alfa`, `bravo`, `charlie` ((based on standard phonetic alphabet terms)) when I drafted the tests, but finally I just created only one and left the name as `alfa`. It looks unusual to use `alfa` here and I changed it to `test-policy`.